addon/models/osf-model.js

```diff
@@ -20,20 +20,25 @@ export default DS.Model.extend(HasManyQuery.ModelMixin, {
         this._super(...arguments);
         this.set('_dirtyRelationships', Ember.Object.create({}));
     },
+    resolveRelationship(rel) {
```

Comment:
Yes - this code was used in the save method but it was pulled out to be a method on its own. This method was needed in the osf-adapter. The method returns a hasMany or BelongsToRelationship object that the addCanonicalRecord should be called on.  

I'll add the docstring!